src/sentry/api/endpoints/project_metrics_usage.py

```diff
@@ -0,0 +1,98 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers.rest_framework.dashboard import DashboardWidgetQuerySerializer
+from sentry.incidents.models.alert_rule import AlertRule
+from sentry.models.dashboard_widget import DashboardWidget
+from sentry.models.project import Project
+from sentry.sentry_metrics.models import SpanAttributeExtractionRuleConfig
+
+
+class MetricsUsageAlertSerializer(serializers.Serializer):
```

Comment:
Serializers should be located with the others at https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/metrics_extraction_rules.py or in the appropriate file w.r.t. widgets/alerts.

Reply:
Moved